modules/violation-view-policy-tab.adoc

```diff
@@ -9,21 +9,34 @@
 The *Policy* tab of the *Details* panel displays details of the policy that caused the violation.
 
 [discrete]
-== Policy Details section
+== Policy overview section
 
-The policy details section lists the following information:
+The policy overview section lists the following information:
 
-* *Id*: The numerical identifier for the policy.
-* *Name*: The name of the policy.
+* *Severity* - The risk level for the violation.
```

Comment:
It's not really risk. I think we should perhaps match what's listed on the [policy management docs](https://64341--docspreview.netlify.app/openshift-acs/latest/operating/manage-security-policies#use-default-security-policies_manage-security-policies)?

> Severity: A ranking of the policy, either critical, high, medium, or low, for the amount of attention required.